core/amber/src/test/scala/edu/uci/ics/amber/engine/e2e/TestOperators.scala

```diff
@@ -15,11 +15,19 @@ import edu.uci.ics.texera.workflow.operators.source.sql.mysql.MySQLSourceOpDesc
 object TestOperators {
 
   def headerlessSmallCsvScanOpDesc(): CSVScanSourceOpDesc = {
-    getCsvScanOpDesc("src/test/resources/CountrySalesDataHeaderlessSmall.csv", header = false)
+    getCsvScanOpDesc("src/test/resources/country_sales_headerless_small.csv", header = false)
+  }
+
+  def headerlessSmallMultiLineDataCsvScanOpDesc(): CSVScanSourceOpDesc = {
+    getCsvScanOpDesc(
+      "src/test/resources/country_sales_headerless_small_multi_line.csv",
+      header = false,
+      multiLine = true
+    )
   }
 
   def smallCsvScanOpDesc(): CSVScanSourceOpDesc = {
-    getCsvScanOpDesc("src/test/resources/CountrySalesDataSmall.csv", header = true)
+    getCsvScanOpDesc("src/test/resources/country_sales_small.csv", header = true)
```

Comment:
I think adding data files to the repo can increase the size and spam it maybe you can generate a dummy data and write it in a file then test reading it then delete the file upon test end.

Reply:
those are generated dummy data used for testing purposes. Those are test files that are already in the repo. I agree if we can generate them in testing runtime would be great, but that's an overhead for now IMO.